### DISCUSSION TITLE
change `CONFIG_ESP_CONSOLE_UART_BAUDRATE` can't valid

### DISCUSSION THREAD

**Discussion by xiaguangbo** [esp-rs/esp-template#171] [Q&A]
chip: esp32c2
XTAL: 26M
defualt monitor baudrate: 74880

Add `CONFIG_ESP_CONSOLE_UART_BAUDRATE=115200` in `sdkconfig.defaults`, delete `target/riscv32imc-esp-espidf/release/build/esp-idf-sys-...`, rebuild.
Search `CONFIG_ESP_CONSOLE_UART_BAUDRATE` in `target/riscv32imc-esp-espidf/release/build/esp-idf-sys-...`, but result is still 74880.

**Comment by xiaguangbo** [comment]
These are valid
```
CONFIG_XTAL_FREQ_26=y
CONFIG_FREERTOS_HZ=1000
CONFIG_ESP_TASK_WDT_EN=n
```

**Comment by bjoernQ** [comment]
Seems you are using Rust on ESP-IDF - this repository is the template for esp-hal (bare-metal). Please ask in https://github.com/esp-rs/esp-idf-template

Going to close this since it's unrelated to bare-metal Rust on ESP32
